en/snap/advanced-feature.md

```diff
@@ -551,30 +560,31 @@ curl -X POST \
 ```
 <!-- tabs:end -->
 
-Be sure to use the same `user_id` for that particular customer on future transactions. Their card will be previewed on Snap card payment page, on future transactions.
+Use the same `user_id` for that particular customer on future transactions. Their card will be previewed on Snap card payment page, on future transactions.
 
 ![enabled payment card](./../../asset/image/snap-adv-save-card-preview.jpg)
 
-To better understand the use cases, you an also further refer to [this article](https://support.midtrans.com/hc/en-us/articles/360002419153-One-Click-Two-Clicks-and-Recurring-Transaction).
+For more use cases, refer to [One Click, Two Click, and Recurring Transaction](https://support.midtrans.com/hc/en-us/articles/360002419153-One-Click-Two-Clicks-and-Recurring-Transaction).
 
 ### Recurring / Subscription Card Transaction
-Snap can be utilized **to initialize** subscription or recurring payment flow. Note that:
+Snap can be utilized to initialize subscription or recurring payment flow. Note that:
 
 * You will require [Core API](/en/core-api/overview.md) to do the recurring charge.
-* Currently, recurring only support card transaction.
+* Currently, recurring transaction supports only card transactions.
 
-Please refer to sequence below to understand the recommended flow:
+Please refer to the sequence diagram given below to understand the recommended flow.
 
 <input id="seq-diag" class="collaps-toggle" type="checkbox">
 <label for="seq-diag" class="collaps-label"><strong>Snap Recurring Sequence Diagram</strong></label>
 <div class="collaps-content">
 
 ![snap recurring sequence](./../../asset/image/snap-adv-snap-recurring-sequence.png)
-</div> 
+</div>
 
-During the first transaction, example of the JSON param (this param is used during [API Request Step](/en/snap/integration-guide.md#api-request)):
+Example of the JSON parameters used during  [API Request Step](/en/snap/integration-guide.md#api-request) is given below.
```

Comment:
You were removing some important words there, this sentence should says the example is intended **for first/initial transaction**. Because if you study the diagram, Snap is only used for the first/initial transaction.